.github/workflows/staging_deploy.yml

```diff
@@ -1,4 +1,4 @@
-name: Deploy to EC2 staging
+name: Deploy to EC2 Staging
 
 on:
   push:
```

Comment:
Shouldn't we also deploy to staging when a PR is made?

Reply:
We only deploy to staging when changes are merged into the main branch to prevent any unintended migrations from running on the staging db.